src/rtmessage/rtrouted.c

```diff
@@ -1699,7 +1699,8 @@ rtRouted_AcceptClientConnection(rtListener* listener)
   }
 
   uint32_t one = 1;
-  setsockopt(fd, SOL_TCP, TCP_NODELAY, &one, sizeof(one));
+  if (setsockopt(fd, SOL_TCP, TCP_NODELAY, &one, sizeof(one)) == -1)
```

Comment:
The change is good but there is a bug exist here where the socket that is accepted could be unix socket and we mustn't set SOL_TCP.

There is already a ticket to address this and we have Fix already being tested.

Thanks,